styles.css

```diff
@@ -0,0 +1,108 @@
+:root {
+  --icon-font-family: 'Poppins';
+}
+
+body {
+  background-color: black;
+}
+
+.container {
+  display: flex;
+  border: 1px solid blue;
+  width: 375px;
+  height: 100vh;
+  flex-wrap: wrap;
+  flex-direction: column;
+  background-image: url("images/header-shapes mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+  background-color: white;
+}
+
+.headline {
+  height: auto;
+  width: auto;
+  margin-top: 120px;
+}
```

Comment:
- Your headline class inside headline section should use flexbox to be able to control the contents inside.
- Follow the figma design for fonts, font-size, and color. The margin/padding should feel similar to the figma design.